### PR TITLE
[DEV APPROVED] Updates usage instruction

### PR DESCRIPTION
**Pull request by Tr4pSt3R**
This PR corrects some initial assumptions we had about the FinCap application. I am updating the README accordingly to communicate the rectifications.

### CODE REVIEW

**Comment by Tr4pSt3R** [comment]
♻️ @tomas-stefano

**Comment by margOnline** [comment]
I'm wondering if its best to not specify a port as we all have different set ups?

**Comment by Tr4pSt3R** [comment]
@margOnline we are specifying a port as it's likely to clash with the instructions within the `.env-example` file where the port number for CMS is set to 3000. This is the default port number when we run `rails s`. This is why we specify a port number. 

_Do you get what I mean?_ Do you have any suggestions please @margOnline ? 

https://github.com/moneyadviceservice/fin_cap/blob/a0899bb0ded76175290bc6409ba07d7c1d6bd738/.env-example#L1-L2

**Comment by margOnline** [comment]
I understand that the 2 apps need to be started on different ports. However the instructions in the readme in this pr, _line 16_, read to me as if the cms should be started on port 5000 and the fincap app should be started on port 5000. Since people tend to have their preferred setups I would be inclined not to specify a port and **emphasise** that the 2 apps need to be started on **different** ports. This is a minor non-blocking comment.

**Comment by Tr4pSt3R** [comment]
@margOnline ♻️ 
@tomas-stefano ♻️

File: README.md [hunk 1]
```diff
@@ -27,30 +27,34 @@ $ bundle exec rake db:create \
...
-This is an engine used by the Frontend site and requires data from the [MAS CMS](https://github.com/moneyadviceservice/cms). Follow the instructions below to start the engine via the Frontend. You may also need to refer to [the Frontend README](https://github.com/moneyadviceservice/frontend/blob/master/README.md) if you have not already set up the Frontend site to run locally.
+This is a standalone application  and requires data from the [MAS CMS](https://github.com/moneyadviceservice/cms). Follow the instructions below to get started.
 
   1. Start the CMS
   Navigate to your local CMS directory and start the application, e.g.
 
   ```sh
   $ rails s
   ```
+  Ensure that the FinCap application and the CMS application runs on different
```
Comment (margOnline) [inline README.md]: `run`

**Comment by margOnline** [comment]
👍

**Pull request merged**